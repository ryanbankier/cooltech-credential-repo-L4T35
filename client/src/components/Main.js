import React, { Component } from "react";

// import components
import Search from './Search'
import Header from './Header'
//import boostrap
import Container from 'react-bootstrap/Container'
import Row from 'react-bootstrap/Row'


class Main extends Component{
    constructor(props) {
        super (props)
    }
    render(){

        return(
            <>
            <Container fluid>
                <Row>
                    <Header/>
                </Row>
                <Row >
                    <Search/>
                </Row>
            </Container>
            </>
        )
    }
}

export default Main