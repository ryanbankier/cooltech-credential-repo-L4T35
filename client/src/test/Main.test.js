// a snapshot test of the UI
import React from 'react';
import Main from '../components/Main';
import renderer from 'react-test-renderer';

test('renders correctly', () => {
    const tree = renderer
    .create(<Main/>)
    .toJSON();
    expect(tree).toMatchSnapshot();
    });