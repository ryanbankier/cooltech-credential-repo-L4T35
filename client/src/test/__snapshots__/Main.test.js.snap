// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="container-fluid"
>
  <div
    className="row"
  >
    <header
      className="header-container"
    >
      <div
        className="container-fluid"
      >
        <h1>
          Git Repositories Search App
        </h1>
      </div>
    </header>
  </div>
  <div
    className="row"
  >
    <div
      className="container"
    >
      <form
        className=""
      >
        <div
          className="justify-content-md-center row row-cols-md-4"
        >
          <div
            className="col-md-3"
          >
            <div>
              <input
                className="form-control"
                id="search"
                name="search"
                onChange={[Function]}
                type="text"
              />
            </div>
          </div>
          <div
            className="col"
          >
            <button
              className="btn btn-primary"
              disabled={false}
              onClick={[Function]}
              type="submit"
            >
              Search
            </button>
          </div>
        </div>
      </form>
      <div>
        <div
          className="container"
        >
          <h2>
            Git repositories
          </h2>
          <div
            className="row"
          >
            <div
              className="col-md-2"
            >
              <div
                className="list-group list-group-flush"
                onKeyDown={[Function]}
              >
                <h4>
                  <img
                    alt="Github logo"
                    className="d-inline-block align-top"
                    height="30"
                    src="./images/GitHub.png"
                  />
                   GitHub Users
                </h4>
                <div
                  className="scrollTogg1 overflow-auto"
                  id="xpCol"
                />
              </div>
              <br />
              <div
                className="list-group list-group-flush"
                onKeyDown={[Function]}
              >
                <h4>
                  <img
                    alt="Gitlab logo"
                    className="d-inline-block align-top"
                    height="30"
                    src="./images/gitlab.png"
                  />
                   GitLab Users
                </h4>
                <div
                  className="scrollTogg1 overflow-auto"
                  id="xpCol"
                />
              </div>
            </div>
            <div
              className="col-md-10"
            >
              <div
                className="row"
              >
                <div
                  className="bio col-md-3"
                />
                <div
                  className="repo col"
                >
                  <h3>
                    Welcome Git Repository Search App
                  </h3>
                  <p>
                    Search your for users on both Github and GitLab
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
